deepmd/pd/infer/deep_eval.py

```
@@ -0,0 +1,742 @@
+        if cells is not None:
+            assert isinstance(cells, paddle.Tensor), err_msg
+        if spins is not None:
+            assert isinstance(spins, paddle.Tensor), err_msg
+        assert isinstance(atom_types, paddle.Tensor) or isinstance(atom_types, list)
+        atom_types = paddle.to_tensor(atom_types, dtype=paddle.int64).to(DEVICE)
+    elif isinstance(coords, np.ndarray):
+        if cells is not None:
+            assert isinstance(cells, np.ndarray), err_msg
+        if spins is not None:
+            assert isinstance(spins, np.ndarray), err_msg
+        assert isinstance(atom_types, np.ndarray) or isinstance(atom_types, list)
```

Ruff

Multiple `isinstance` calls for `atom_types`, merge into a single call

Merge `isinstance` calls for `atom_types`

(SIM101)